client/src/Components/Spectrogram/SpectrogramPage.tsx

```diff
@@ -49,10 +50,7 @@ export const SpectrogramPage = () => {
   const { type, account, container, filePath, sasToken } = useParams();
 
   const handleNewSlice = (newSlice: IQDataSlice) => {
-    setDownloadedTiles((oldTiles) => {
-      oldTiles.push(newSlice.index);
-      return oldTiles;
-    });
+    console.log('newSlice', newSlice);
```

Comment:
do we need this

Reply:
Temoved that and also the function call on success because it was deprecated.